remo/dashboard/views.py

```diff
@@ -241,7 +242,7 @@ def list_action_items(request):
     action_items = action_items.distinct()
     number_of_action_items = action_items.count()
 
-    sort_key = request.GET.get('sort_key', LIST_ACTION_ITEMS_DEFAULT_SORT)
+    sort_key = request.GET.get('sort_key', settings.ITEMS_PER_PAGE)
```

Comment:
Good catch. No, the LIST_ACTION_ITEMS_PER_PAGE is what needs replacement. Fixed.